presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaMetadata.java

```diff
@@ -95,6 +97,14 @@
                     .column("partition_key", createUnboundedVarcharType())
                     .column("partition_value", createUnboundedVarcharType())
                     .build())
+            .table(tableMetadataBuilder(TABLE_TABLE_PRIVILEGES)
```

Comment:
yeah, I can't find that information in the `PrivilegeGrantInfo` object in the Hive metastore API.